components/net/bluetooth_thread.rs

```diff
@@ -115,6 +119,24 @@ fn matches_filter(device: &BluetoothDevice, filter: &BluetoothScanfilter) -> boo
             }
         }
     }
+
+// Step 4.
+// TODO: Implement get_manufacturer_data to device.
+//    if let Some(manufacturer_id) = filter.get_manufacturer_id() {
+//        if !device.get_manufacturer_data().contains_key(manufacturer_id) {
+//            return false;
+//        }
+//    }
+//
+// Step 5.
+// TODO: Implement get_device_data to device.
```

Comment:
`Implement get_device_data to device.` -> `Implement get_device_data in device crate.`